### PR TITLE
(#216) - Handle status in ProxyBlob.content

### PR DESCRIPTION
**Pull request by andreoss**
Per #216 
- Check `status`, throw `ArtipieHttpException` if status is not 200, otherwise return content

### CODE REVIEW

File: src/main/java/com/artipie/docker/proxy/ProxyBlob.java [hunk 1]
```diff
@@ -88,16 +86,24 @@ public Digest digest() {
...
+                            Flowable.fromPublisher(body)
+                                .doOnError(terminated::completeExceptionally)
+                                .doOnTerminate(() -> terminated.complete(null))
+                        )
+                    );
+                    sent = terminated;
+                } else {
+                    sent = new FailedCompletionStage<Void>(new IllegalStateException(
+                        String.format("Unexpected status: %s", status)
+                    )).toCompletableFuture();
```
Comment (g4s8) [inline src/main/java/com/artipie/docker/proxy/ProxyBlob.java:104]: Let's use `ArtipieHttpException` here: https://github.com/artipie/http/blob/master/src/main/java/com/artipie/http/ArtipieHttpException.java

**Review by g4s8 — CHANGES_REQUESTED** [review]
@andreoss looks good, please check one comment. Also, please fix PR description: add a short description of your solution to put it into the git history log.

File: src/main/java/com/artipie/docker/proxy/ProxyBlob.java (hunk 1 above)
Comment (andreoss) [reply src/main/java/com/artipie/docker/proxy/ProxyBlob.java]: @g4s8 Done

**Comment by kirill-h5i** [comment]
@andreoss please check CI logs, it fails

**Comment by andreoss** [comment]
@kirill-h5i Done, sorry for the delay

**Review by g4s8 — APPROVED** [review]
@andreoss thanks

**Pull request merged**